common/params_pyx.pyx

```diff
@@ -105,7 +117,7 @@ cdef class Params:
     return self.p.allKeys()
 
 def put_nonblocking(key, val, d=""):
-  threading.Thread(target=lambda: Params(d).put(key, val)).start()
+  Params(d).put_non_blocking(key, val)
 
 def put_bool_nonblocking(key, bool val, d=""):
-  threading.Thread(target=lambda: Params(d).put_bool(key, val)).start()
+  Params(d).put_bool_non_blocking(key, val)
```

Comment:
shall we also remove these helpers?

Reply:
test_power_monitor.py still needs this to mock it to a block writing.

Reply:
fixed that in https://github.com/commaai/openpilot/pull/29788